src/refinement-list/refinement-list.ts

```diff
@@ -1,32 +1,15 @@
 import { Component, Input, Inject, forwardRef, Optional } from '@angular/core';
 import { connectRefinementList } from 'instantsearch.js/es/connectors';
-import { BaseWidget } from '../base-widget';
-import {
-  NgAisInstantSearch,
-  FacetSortByStringOptions,
-} from '../instantsearch/instantsearch';
+import { TypedBaseWidget } from '../typed-base-widget';
+import { NgAisInstantSearch } from '../instantsearch/instantsearch';
```

Comment:
this diff makes it a bit hard to see, I first thought NgAisInstantSearch was a new import here but it's just moved